### PR TITLE
[ticket/15863] Add core.viewforum_modify_sort_direction

### PR DESCRIPTION
**Pull request by 3D-I**
PHPBB3-15863

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15863

### CODE REVIEW

**Comment by 3D-I** [comment]
!set Event

**Comment by 3D-I** [comment]
I don't know the reason of such failure.

**Comment by kasimi** [comment]
The trailing comma is missing: `	'direction',`

**Comment by 3D-I** [comment]
Without a comma on a line and with a comma otherwise, both correct... a little confusing.

**Comment by kasimi** [comment]
That's what phpBB's event exporter expects.

File: phpBB/viewforum.php [hunk 1]
```diff
@@ -598,6 +598,18 @@
 	$sql_start = $start;
 }
 
+/**
+ * Modify the topics sort ordering if needed
+ *
+ * @event core.viewforum_modify_sort_direction
+ * @var string	direction	Topics sort order
+ * @since 3.2.4
```
Comment (marc1706) [inline phpBB/viewforum.php]: Please change the target version to `3.2.5-RC1` as we won't be accepting any new events into the 3.2.4 release.

Comment (3D-I) [reply phpBB/viewforum.php]: Will do

**Comment by marc1706** [comment]
Rebased on 3.2.x and squashed commits.

**Pull request merged**